python/paddle/v2/fluid/io.py

```diff
@@ -11,6 +11,16 @@
 
 
 def is_parameter(var):
+    """Check whether the variable is a Parameter
+
+    This function checks whether the input variable is a Parameter.
+
+    Args:
+        var : The input variable.
+
+    Returns:
+        boolean result whether the variable is a Parameter
```

Comment:
Is line 14 and line 22 lacking periods in the end?

Reply:
Thank you! I will fix these things. Just wanted to check if it renders the comment at all.

Reply:
Fixed.